src/styles/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background-image: url('../images/star-active.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    display: flex;
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+
+    &:nth-child(5) {
```

Comment:
I've mentioned this before, but try changing this approach so you don't have fixed styles that forces all products to be 4 stars active